install_config/install/rpm_vs_containerized.adoc

```diff
@@ -13,29 +13,34 @@ toc::[]
 
 == RPM vs Containerized Installation
 
-An RPM installation installs all services through package management and
-configures services to run within the same user space, while a containerized
-installation installs services using container images and runs separate services
-in individual containers.
+You can opt to install {product-title} using the RPM or containerized package
+method. Either installation method results in a working environment, but the
+choice comes from the operating system and how you choose to update your hosts.
 
+[IMPORTANT]
+====
 The default method for installing {product-title} on
 ifdef::openshift-origin[]
 Fedora, CentOS, or RHEL
 endif::[]
 ifdef::openshift-enterprise[]
 Red Hat Enterprise Linux (RHEL)
 endif::[]
-uses RPMs.
-
-[IMPORTANT]
-====
-When targeting a Red Hat Atomic Host system, the containerized method is the
-only available option, and is automatically selected for you based on the
-detection of the *_/run/ostree-booted_* file.
+uses RPMs. When targeting a Red Hat Atomic Host system, the
+containerized method is the only available option, and is automatically selected
+for you based on the detection of the *_/run/ostree-booted_* file.
 ====
 
-The following table outlines the differences between the RPM and Containerized
-methods:
+When using RPMs, all services are installed and updated via package management
+from an outside source. These modify a host's existing configuration within the
+same user space. Alternatively, containerized installs instead are a complete,
+all-in-one resource using container images and its own operating system within
+the container. Any updated, newer containers replace any existing ones on your
+host. Choosing one method over the other depends on how you choose to update
+your packages in the future.
```

Comment:
Replace: "Choosing one method over the other depends on how you choose to update your packages in the future."

With: "Choosing one method over the other depends on how you choose to update {product-title} in the future."

Rationale: Containerized installation isn't package based.